python/tk_multi_workfiles/file_finder.py

```diff
@@ -280,9 +273,7 @@ def _process_work_files(
 
             # copy common fields from work_file:
             #
-            file_details = dict(
-                [(k, v) for k, v in six.iteritems(work_file) if k != "path"]
-            )
+            file_details = dict([(k, v) for k, v in work_file.items() if k != "path"])
```

Comment:
Will it work?

```suggestion
            file_details = {k, v for k, v in work_file.items() if k != "path"}
```

Same question for the rest of this file.